### ISSUE TITLE
Section 3.1

### ISSUE THREAD

**Issue by karenlmasters** [tingard/gz-builder-method-paper#56]
Needs a bit of reorganizing and more of a lead in. I would suggest. 

The calibration subset was a set of nine synthetic galaxy images created from Galaxy Builder models, which were then re-run through the Galaxy Builder process. These galaxies were used to fine-tune clustering and fitting hyperparameters (See Section XX).  Our ability to recover morphology accurately is essential validation for our ability to recover good photometic models of galaxies. 

 Overall, these results highlight the importance of good priors to obtain accurate fits of complex photometric models. In more detail, the models recovered for the nine synthetic galaxy images demonstrate that: 

 1. We successfully recover all spiral arms present, and do not receive any false positives
 2. The spiral pitch angles obtained through aggregation vary by <9deg  from the true values, with fitting improving this error slightly.
 3.  Volunteers systematically use elongated bulges to fit for bar components. This resulted in two false positives for bulge presence in the aggregate models. This feature (switching light between model components) is a common issue in all photometric fitting methods. 
 4. The Jaccard metric is unstable to small changes in rotation for highly elliptical components (i.e. bars). This resulted in one false negative of bar presence in the aggregate model. 

 The fitting step for this subset of images highlighted the benefit of obtaining a rough starting point through clustering of user classifications. The method struggled to recover structural parameters for which we did not obtain such a starting point (S ́ersic index and bar boxyness). These parameters are degenerate with the total flux in the bulge and bar, and therefore are crucial for measurements of bulge fraction, but difficult to identify using gradient descent (Lackner & Gunn 2012). Reduced Chi-squard values for the fits range from 1.000 to 1.331."

**Comment by karenlmasters** [comment]
Also I'm not sure what you are trying to say with this: 
"therefore are crucial for measurements of bulge fraction, but difficult to identify using gradient descent (Lackner & Gunn 2012). Reduced Chi-squard values for the fits range from 1.000 to 1.331."

What is the reader meant to conclude from this?
